tests/lcs/core/interface/interface.livecodescript

```diff
@@ -823,4 +823,16 @@ type "a"
 
 TestAssert "test", the text of field "fi" is "bar"
 
-end TestInterface101
\ No newline at end of file
+end TestInterface101
+
+
+on TestLinkHiliteColors
+   set the linkhilitecolor to "blue"
+   TestAssert "test1", the linkhilitecolor is "blue"
+
+   set the linkhilitecolor to "255,255,0"
+   TestAssert "test2", the linkhilitecolor is "255,255,0"
+
+   set the linkhilitecolor to "#336633"
+   TestAssert "test3", the linkhilitecolor is "#336633"
+end TestLinkHiliteColors
```

Comment:
The docs states that
`By default, the linkHiliteColor is set to "255,0,0" (bright red). The linkHiliteColor of a newly created stack is set to empty by default.`.
Currently, if you create a new stack, the `linkhilitecolor` is indeed `empty` - but that might not be the case in the tests (needs to be checked I suppose)